### PR TITLE
Add custom time scale to x-axis of line charts

### PR DESCRIPTION
**Pull request by juliawu**
Allows for selection of a custom time scale in line web component, and sets the line charts in the UN sdg pages to use only years in the x-axis.

Example:
```
<datacommons-line
  header="Sample line"
  places="country/COL"
  variables="sdg/SI_POV_NAHC"
/>
```
Gives
Screenshot 2023-09-12 at 6 52 14 PM

Instead of
Screenshot 2023-09-12 at 6 52 48 PM

### CODE REVIEW

File: static/js/chart/types.ts [hunk 1]
```diff
@@ -170,6 +170,7 @@ export interface HorizontalBarChartOptions extends ChartOptions {
 
 export interface LineChartOptions extends ChartOptions {
   handleDotClick?: (dotData: DotDataPoint) => void;
+  timeScale?: string;
```
Comment (dwnoble) [inline static/js/chart/types.ts:173]: Use typing `"year" | "month" | "day"` instead of `string`

File: static/js/components/tiles/line_tile.tsx [hunk 2]
```diff
@@ -80,6 +80,8 @@ export interface LineTilePropType {
   showTooltipOnHover?: boolean;
   // Function used to get processed stat var names.
   getProcessedSVNameFn?: (name: string) => string;
+  // Time scale to use on x-axis. "year", "month", or "day"
+  timeScale?: string;
```
Comment (dwnoble) [inline static/js/components/tiles/line_tile.tsx:84]: same as above

File: static/library/line_chart_component.ts [hunk 3]
```diff
@@ -111,6 +111,10 @@ export class DatacommonsLineComponent extends LitElement {
   @property({ type: Boolean })
   showExploreMore: boolean;
 
+  // Optional: Timescale to use for x-axis labels
+  @property()
+  timeScale?: string;
```
Comment (dwnoble) [inline static/library/line_chart_component.ts:116]: ditto

File: static/js/chart/types.ts (hunk 1 above)
Comment (juliawu) [reply static/js/chart/types.ts]: Good call! Done!

File: static/js/components/tiles/line_tile.tsx (hunk 2 above)
Comment (juliawu) [reply static/js/components/tiles/line_tile.tsx]: Done.

File: static/library/line_chart_component.ts (hunk 3 above)
Comment (juliawu) [reply static/library/line_chart_component.ts]: Done.

**Review by dwnoble — APPROVED** [review]

**Pull request merged**